Kudu.Services/Jobs/JobsController.cs

```diff
@@ -179,6 +179,12 @@ public HttpResponseMessage InvokeTriggeredJob(string jobName, string arguments =
         {
             try
             {
+                if (FileSystemHelpers.IsFileSystemReadOnly())
```

Comment:
Could this be moved to a `catch` block instead, so it doesn't affect the happy path?